### DISCUSSION TITLE
WebAssembly?

### DISCUSSION THREAD

**Discussion by joex92** [melonjs/melonJS#1173] [General]
Does MelonJS uses or implements WebAssembly or WASM in the code?

**Comment by obiot** [comment]
not yet ! We discussed a few times about at least using it for Vector and Matrix operations, but there are so many things to do and only a few people working on melonJS, that nobody never really had the time to properly look at it.

**Reply by joex92** [reply]
oh ok, I was wondering because I was asking chatGPT about js libraries using WebAssembly, and it told me that this library was implemented on WebAssembly 😅
but I didn't know about this library before, I might still give it a go...

**Comment by obiot** [comment]
that's interesting ! we did have some intense talks about it at some point, maybe ChatGPT is confused with the discussion we had back then on the old forum, chat, etc...  dunno, but that's funny :)
